portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase

```diff
@@ -46,6 +46,102 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-176997. TC-1: Verify MALU, APV and Scope are enabled on default instance when a LXC environment is being set up."
+	@priority = 5
+	test AssertAuditHeadersOnDefaultInstance {
+		property liferay.online.properties = "true";
+		property test.liferay.virtual.instance = "false";
+
+		task ("When: Get HTTP response headers in login state") {
+			var curl = '''http://localhost:8080/web/guest?SM_USER=test@liferay.com -I''';
+
+			var response = JSONCurlUtil.post(${curl});
+		}
+
+		task ("Then: Assert the values of X-Liferay-Request-Company and X-Liferay-Request-Group are companyId and groupId respectively and X-Liferay-Request-Guest-User is false.") {
+			TestUtils.assertPartialEquals(
+				actual = ${response},
+				expected = "X-Liferay-Request-Company: 20096");
```

Comment:
The companyId will change after a period of time, try not to write a fixed value, please use JSON to get values.